numpy/lib/_npyio_impl.py

```diff
@@ -526,7 +526,7 @@ def save(file, arr, allow_pickle=True, fix_imports=np._NoValue):
 
         .. deprecated:: 2.1
             This flag is ignored since NumPy 1.17 and was only needed to
-            support loading some files in Python 2 written in Python 3.
+            support loading in Python 2 some files written in Python 3.
```

Comment:
I don't see why this needs to change

Reply:
I'm not a native speaker, of course, but I find it much easier to understand this way.